hearpreprocess/pipeline.py

```diff
@@ -681,8 +681,9 @@ class MetadataTask(WorkTask):
     def metadata(self):
         if self._metadata is None:
             self._metadata = pd.read_csv(
-                self.metadata_task.workdir.joinpath(self.metadata_task.outfile)
-            ).astype(str)
+                self.metadata_task.workdir.joinpath(self.metadata_task.outfile),
+                dtype={"relpath": str, "unique_filestem": str, "split": str},
```

Comment:
Do we want label to be str as well? For maestro the labels are interpreted as int and that breaks downstream - so those need to be str before downstream.

Reply:
since the label is float for `maestro`, that will automatically be typed by pandas while reading.